### ISSUE TITLE
functions reminder tutorial

### ISSUE THREAD

**Issue by brownsarahm** [brownsarahm/python-data-project#21]
in [packaging episode](https://github.com/brownsarahm/python-data-project/blob/gh-pages/_episodes/03-packaging-installing.md)

**Comment by rra94** [comment]
Hi I can take this. Just to confirm. I have to write how a package works in python right?

**Comment by brownsarahm** [comment]
yes, write abouot how it works, in language for a beginner to the topic, with example code
